apps/wpcom-block-editor/src/calypso/iframe-bridge-server.js

```diff
@@ -538,7 +538,8 @@ function handleGoToAllPosts( calypsoPort ) {
  */
 function openLinksInParentFrame() {
 	const viewPostLinkSelectors = [
-		'.components-notice-list .is-success .components-notice__action.is-link', // View Post link in success notice
+		'.components-notice-list .is-success .components-notice__action.is-link', // View Post link in success notice, Gutenberg <5.9
+		'.components-notice-list .components-snackbar__content a', // View Post link in success snackbar, Gutenberg >=5.9
```

Comment:
Close, the `.components-notice-list` also changed. So this should be updated to .components-snackbar-list, or maybe simplified to `.components-snackbar__content a`

<img width="854" alt="Screen Shot 2019-06-18 at 2 12 27 PM" src="https://user-images.githubusercontent.com/1270189/59720211-236d3f80-91d3-11e9-9f08-a794d9dc9e03.png">

Reply:
Ha, yep, just took a long time to figure that out myself 🙃 (https://github.com/Automattic/wp-calypso/pull/34104/commits/107e7c832921621fa7b686f751e45aa014557658)